File: src/main/java/com/fengsheng/handler/add_order_tos.java
package com.fengsheng.handler;

import com.fengsheng.HumanPlayer;
import com.fengsheng.Statistics;
import com.fengsheng.protos.Fengsheng;

public class add_order_tos extends AbstractProtoHandler<Fengsheng.add_order_tos> {
    @Override
    protected void handle0(HumanPlayer r, Fengsheng.add_order_tos pb) {
        Statistics.getInstance().addOrder(r.getDevice(), r.getPlayerName(), pb.getTime());
        r.send(Fengsheng.add_order_toc.getDefaultInstance());
    }
}

File: src/main/kotlin/handler/add_order_tos.kt
package com.fengsheng.handler

import com.fengsheng.HumanPlayer
import com.fengsheng.protos.Fengsheng

class add_order_tos : AbstractProtoHandler<Fengsheng.add_order_tos>() {
    override fun handle0(r: HumanPlayer, pb: Fengsheng.add_order_tos) {
        r.send(Fengsheng.add_order_toc.getDefaultInstance())
    }
}